core/src/test/java/org/apache/iceberg/TestSortOrder.java

```diff
@@ -342,6 +342,22 @@ public void testVariantUnsupported() {
         .hasMessage("Unsupported type for identity: variant");
   }
 
+  @TestTemplate
```

Comment:
```suggestion
  @Test
```
This maybe a general question for other tests in `TestSortOrder`. Do we need `TestTemplate` for tests that does not use the parametrized field, `formatVersion`. It seems we're running the same test three times.

Reply:
Good call! Some tests use the `formatVersion` indeed. I've limited it to the `unknown` related test for now